src/index.html

```diff
@@ -11,6 +11,21 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="1" method="POST"
+    data-qa="big" class="form">
+      <img src="./images/Search.svg"
+      alt="Search"
+      class="icon">
+      <input type="text" class="form__input"
+      placeholder="Try &quot;Los Angeles&quot;">
+    </form>
+    <form action="1" method="POST"
+    data-qa="small" class="form">
+      <img src="./images/Search.svg"
+      alt="Search"
+      class="icon-small">
+      <input type="text" class="form__input"
+      placeholder="Try &quot;Los Angeles&quot;">
+    </form>
```

Comment:
code it not aligned, please check the checklist 
<img width="1133" alt="Screenshot 2023-09-19 at 13 03 48" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64746943/606697c0-0a0b-45a2-b8ae-856711e602cb">